wcomponents-core/src/main/java/com/github/bordertech/wcomponents/render/webxml/WTextAreaRenderer.java

```diff
@@ -60,7 +58,10 @@ public void doRender(final WComponent component, final WebXmlRenderContext rende
 			xml.append("<ui:rtf />");
 		}
 
-		xml.appendEscaped(textArea.getText());
+		String textString = textArea.getText();

```

Comment:
This has nothing to do with malicious HTML or script. It unescapes HTML character entities such as &bul; back to their character. These entities are not permitted in XML whereas the Characters are because we serve it'd-8. This was the proper fix we should have done years ago instead of putting in a doctype and telling people not to use these entities.